### ISSUE TITLE
Selenium update breaking selenider_session()

### ISSUE THREAD

**Issue by AntonioBergallo** [ashbythorpe/selenider#27]
Hey! I was trying to run selenider now and got this error message 

session_sel <- selenider_session(
session = "selenium",
browser = "chrome",
timeout = 6)

Error in utils::download.file(url, path, quiet = !verbose) : 
  cannot open URL 'https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.23.0/selenium-server-4.23.0.jar'
In addition: Warning messages:
1: In utils::download.file(url, path, quiet = !verbose) :
  downloaded length 0 != reported length 0
2: In utils::download.file(url, path, quiet = !verbose) :
  cannot open URL 'https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.23.0/selenium-server-4.23.0.jar': HTTP status was '404 Not Found'
  
  I think it has somehthing to do with selenium's new release, which has a different name due to being a beta version, as the image below shows.
  
![image](https://github.com/user-attachments/assets/15e6c7fb-c858-4402-8013-23ab6c1a777b)

Is there a quick fix for that? I skimmed through the code for selenider_session but could not find it right away.

**Comment by Doozy134** [comment]
Is this still an issue after v4.23.0 was released 7 days ago?

**Comment by AntonioBergallo** [comment]
No, right now it is working. It might break when a later beta version is released with a name that doesn't fit the pattern. You can also download 4.23.0 and pull the file from your computer. The function itself has a parameter for that, "temp" if I'm not mistaken.

**Comment by ashbythorpe** [comment]
This is an issue in selenium, not selenider, that should be fixed in the dev version of selenium (`pak::pak("ashbythorpe/selenium-r")`). I haven't tested this very thoroughly because I'm in another country at the moment, so let me know if it breaks again.
